### PR TITLE
🐛 Fix the scoring when scaning with inventory packs

### PR DESCRIPTION
**Pull request by czunker**
The score for inventory packs is now unscored and no longer error.

Fixes: #482

### CODE REVIEW

**Comment by czunker** [comment]
Now, it looks like this:
```
... 215 more lines ...



Scanned 1 assets

Ubuntu 22.04.2 LTS
    U X1

For detailed output, run this scan with "-o full".
```

File: policy/executor/internal/nodes.go [hunk 1]
```diff
@@ -530,22 +530,27 @@ func (nodeData *ReportingJobNodeData) score() (*policy.Score, error) {
 		return s, nil
 	}
 
-	calculator, err := policy.NewScoreCalculator(nodeData.scoringSystem)
-	if err != nil {
-		return nil, err
+	s := &policy.Score{
+		Type: policy.ScoreType_Unscored,
```
Comment (preslavgerchev) [inline policy/executor/internal/nodes.go]: Do we need to fill in the other props, such as `datacompletion` and `datatotal` here?

Edit: looks like those values seem to be correct in the db after the scan

**Comment by preslavgerchev** [comment]
I just tested this out too, works great!

File: policy/executor/internal/nodes.go (hunk 1 above)
Comment (czunker) [reply policy/executor/internal/nodes.go]: Good catch. This lead to an even simpler solution. We can re-use the avg calculator. That also takes care of the rest.

**Review by preslavgerchev — APPROVED** [review]

**Pull request merged**